files/en-us/web/api/mediastream/index.html

```diff
@@ -12,7 +12,7 @@
 ---
 <div>{{APIRef("Media Capture and Streams")}}</div>
 
-<p><span class="seoSummary">The <strong><code>MediaStream</code></strong> interface represents a stream of media content. A stream consists of several <strong>tracks</strong> such as video or audio tracks. Each track is specified as an instance of {{domxref("MediaStreamTrack")}}.</span>You can obtain a MediaStream object either by using the constructor or by calling {{domxref("MediaDevices.getUserMedia()")}}.</p>
+<p><span class="seoSummary">The <strong><code>MediaStream</code></strong> interface represents a stream of media content. A stream consists of several <strong>tracks</strong> such as video or audio tracks. Each track is specified as an instance of {{domxref("MediaStreamTrack")}}.</span>You can obtain a MediaStream object either by using the constructor or by calling functions such as {{domxref("MediaDevices.getUserMedia()")}}, {{domxref("MediaDevices.getDisplayMedia()")}}, or {{domxref("HTMLCanvasElement.captureStream()")}}.</p>
```

Comment:
```suggestion
<p>The <strong><code>MediaStream</code></strong> interface represents a stream of media content. A stream consists of several <strong>tracks</strong>, such as video or audio tracks. Each track is specified as an instance of {{domxref("MediaStreamTrack")}}.</p>

<p>You can obtain a <code>MediaStream</code> object either by using the constructor or by calling functions such as {{domxref("MediaDevices.getUserMedia()")}}, {{domxref("MediaDevices.getDisplayMedia()")}}, or {{domxref("HTMLCanvasElement.captureStream()")}}.</p>
```
I *think* were planning to remove all `class="seoSummary"` markup. So as long as we’re touching this, we might as well go ahead and remove it here. (The reason for removing it is that we’re not planning to have any equivalent in markdown, once we migrate all the doc sources away from HTML markup and over to markdown. See https://github.com/mdn/content/pull/5894 and https://github.com/mdn/content/issues/5865 and https://github.com/mdn/content/pull/4085 for more info.